src/rgw/rgw_admin.cc

```diff
@@ -6967,7 +6979,7 @@ int main(int argc, const char **argv)
     int r = RGWBucketAdminOp::info(store, bucket_op, stream_flusher, null_yield, dpp());
     if (r < 0) {
       cerr << "failure: " << cpp_strerror(-r) << ": " << err << std::endl;
-      return -r;
+      return -posix_errortrans(r);
```

Comment:
`RGWBucketAdminOp::info()` would be returning `-ERR_NO_SUCH_BUCKET`, so i think this should be:
```c++
return posix_errortrans(-r);
```